src/Native/CMakeLists.txt

```diff
@@ -81,8 +81,8 @@ if (APPLE)
    
    # We cannot enable "stack-protector-strong" on OS X due to a bug in clang compiler (current version 7.0.2)  
    add_compile_options(-fstack-protector)  
-else ()  
-   add_compile_options(-fstack-protector-strong)  
+else ()
+   add_compile_options(-fstack-protector-strong)
```

Comment:
Has anything other than whitespace changed in this file now?  If not, I'd suggest reverting the whole file.